scripts/lint-style.py

```diff
@@ -147,6 +148,13 @@ def set_option_check(lines, path):
                 errors += [(ERR_OPT, line_nr, path)]
     return errors
 
+def windows_line_endings_check(lines, path):
+    errors = []
+    for line_nr, line in enumerate(lines, 1):
+        if b"\r\n" in line:
```

Comment:
with the change I suggest below:
```suggestion
        if "\r\n" in line:
```